libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2479,6 +2479,8 @@ bool TypeChecker::visit(MemberAccess const& _memberAccess)
 				t.front()->category() == Type::Category::Contract)
 			)
 				errorMsg += " Did you intend to call the function?";
+
+
```

Comment:
I guess those got here by accident :-)?

Reply:
Yes, I was adding some debug code, and when deleting didn't delete empty lines as well :D
Fixing now